### ISSUE TITLE
[Bug] Infinite loading on Chrome 95

### ISSUE THREAD

**Issue by eeintech** [darkreader/darkreader#7328]
# Bug Report

## Bug Description
Dark Reader Chrome extension does not load anymore.

## Website Address
All websites

## Steps To Reproduce
Upgrade Chrome to ` 95.0.4638.54`?
Or upgrade Dark reader extension to `4.9.42` on Chrome.
Not sure exactly what broke it.

## Expected Behavior
Loads and functions normally

## Actual Behavior
It loads forever

## Screenshots

![image](https://user-images.githubusercontent.com/4020546/141208637-fae05f67-7d3b-4d7b-88ca-6546650b28f9.png)

## System Information

- OS: Linux Ubuntu 20.04 and Windows 10
- Browser: Chrome 95.0.4638.54
- Dark Reader Version: 4.9.42

## Additional Context
N/A

**Comment by Gusted** [comment]
Hi @eeintech!

Could you try to re-install the dark reader? It might be in some kind of dead-lock.

Regards,
Gusted

**Comment by lyh16** [comment]
I am not sure whether I should file a seperate issue for this, but I'll leave this here for now since the "Steps to Reproduce" are identical.

After updating either Chrome or Dark Reader, Gmail completely broke for me.
I am either stuck at the loading gage bar for Gmail, or the mail list loads but the loading indicator (the spinning blue circle seen next to the tab title) continuously spins; rendering the whole website unusable - I can't click on anything.

The very existence of Dark Reader on Chrome seems to be the problem.
Disabling Dark Reader just for Gmail does not solve the problem.
Forcibly stopping the loading process by clicking on the "X" button next to the back/forward buttons does not solve the problem either.

I have reached this conclusion after completely resetting Chrome and manually reactivating each extension one by one and trying to load Gmail again. The moment Dark Reader is activated, Gmail enters "infinite loading".

I have tried re-installing Dark Reader and it didn't help. This issue occurs only for Gmail, and any other websites are not affected; so far as I have checked.
